worker_node/job/send.js

```diff
@@ -96,8 +99,20 @@ class Send extends Base {
 
     await delay(2 * BLOCK_TIME);
 
-    const writePermission = await this.#ain.db.ref(path).evalRule({value: null});
-    if (!writePermission) {
+    // TODO: update ain-js to support is_global and use ain-js here
+    const response = await axios.post(this.config.ainUrl + 'json-rpc', {
+      method: 'ain_evalRule',
+      params: {
+        ref: path,
+        value: null,
+        address: this.config.ainAddress,
+        is_global: true,
+        protoVer: '0.1.0'
+      },
+      jsonrpc: '2.0',
+      id: 0
+    });
```

Comment:
Suggestion: Could you pass a timeout configuration to prevent infinite wait?

Reply:
Thanks, done!